include/hpcombi/epu8.hpp

```diff
@@ -44,7 +47,14 @@ operator"" _u8(unsigned long long arg) noexcept {  // NOLINT
     return static_cast<uint8_t>(arg);
 }
 
-/// SIMD vector of 16 unsigned bytes
+/**
+epu8 stands for *Extended Packed Unsigned, grouped by 8 bits*;
+this is the low level type chosen by Intel for their API to intrinsics,
+ie a SIMD vector of 16 unsigned bytes (16×8 = 128bits).
+Functions using this type uses semantically equivalent types,
+eg a _m128 which is 2 vect of 64bits.
```

Comment:
"_m128 which is 2 vect of 64bits" is maybe better phrased as "_m128 which is a vector containing 2 signed 64 bits integers"